packages/controllers/src/plugins/PluginController.ts

```diff
@@ -1,30 +1,30 @@
-import { ObservableStore } from '@metamask/obs-store';
-import EventEmitter from '@metamask/safe-event-emitter';
+/* eslint-disable @typescript-eslint/consistent-type-definitions */
```

Comment:
I feel this. We should maybe move it into an override in the ESLint config, or disable the rule on a line-by-line basis if there aren't too many conflicts.

Reply:
Or maybe update `eslint-config` to prefer types over interfaces. Types do seem strictly better these days, because of this incompatibility interfaces have with Record.

Reply:
True, although `interface Foo extends Bar { ... }` is a nice syntax, and sometimes the incompatibility with `Record` isn't a problem 🤔 

Is there a way for a type to extend another type other than through the `&` operator?